SU2_CFD/include/numerics/flow/flow_sources.hpp

```diff
@@ -293,3 +293,57 @@ class CSourceWindGust final : public CSourceBase_Flow {
   ResidualType<> ComputeResidual(const CConfig* config) override;
 
 };
+
+/*!
+ * \class CSourceVolumetricHeat
+ * \brief Class for a source term due to a heat source.
+ * \ingroup SourceDiscr
+ * \author Ruben Sanchez
+ */
+class CSourceVolumetricHeat final : public CSourceBase_Flow {
+private:
+  bool implicit;
```

Comment:
No problem, always happy to learn new stuff. Sorry I moved most classes but that way it is consistent with the other "themes" having 90+90 numerics files might be counterproductive.

By the way I noticed that this class is not used, the integration is done "in place" during the source loop, is that because there are not enough slots in the numerics container? Might be worth leaving a note somewhere.

Reply:
Yes, I agree that it looks much better now with some of the classes together. I didn't know exactly how we wanted to do with the numerics (there are way too many) and I think this is a good compromise between readability and keeping the number of files still manageable.

On the second note, that's a good question. I know for sure I was using this at some point and that I had trouble with the slots in the numerics container. Let me double check a couple of things and I will get back to you after lunch.

Reply:
Indeed, after checking the implementation I remember I moved the integration into ```CIncEulerSolver.cpp``` because there were limited slots, so I stopped using the CSourceVolumetricHeat. I have just removed it to keep the code clean.